DailyReportWeb/app/js/policyEditorApp.js

```diff
@@ -1,6 +1,18 @@
 ﻿angular.module("policyEditorApp", [])
     .controller("policyListPage", ['$scope', '$http', function ($scope, $http) {
-        $http.get("/api/policy").success(function(list){
-            $scope.policyList = list;
-        })        
+    	$http.get("/api/policy").success(function (list) {
+    		$scope.policyList = list;
+    	});
+
+    	$scope.updateReportTime = function (policySelected) {
+    		$http.put("/api/policy",
+				$scope.policyList.filter(function (policy) {
+					return policy == policySelected;
+				})[0]
```

Comment:
Initial gandim altfel solutia in sensul in care trimiteam doar o proprietate si verificam pe care obiect sa se seteze proprietatea - refactored